src/index.html

```diff
@@ -9,6 +9,111 @@
   <link rel="stylesheet" href="style.css">
 </head>
 <body>
-  <h1>Human rights</h1>
+  <h1>Universal Declaration of Human Rights</h1>
+  <h2>Preamble</h2>
+ <p>Whereas recognition of the inherent dignity and of the equal and inalienable
+  rights of all members of the human family is the foundation of freedom, justice
```

Comment:
fix indentations;
when you write some text inside tags move it to the new line and add proper indentations so that the code looks clear